### ISSUE TITLE
ESP32 64 channels

### ISSUE THREAD

**Issue by jeurgenK** [s-marley/ESP32_FFT_VU#15]
Hi Scott,
How are you?

I would like to use the 64 channels. However, the EXCEL list does not help me any further. Do you have any idea ? How should the grading be done? With 32 channels, it looks very nice.

Greetings from Cologne
Juergen

**Comment by s-marley** [comment]
Hi Juergen,

I have updated the spreadsheet to support up to 64 bands. I have no idea if that will work however! With 64 bands, you will see that many frequencies end up in the same bin. To get around this, we nee to increase the number of samples. However, this will slow the update rate of the display. You will have to experiment and see what happens.

Best wishes, Scott.

**Comment by jeurgenK** [comment]
Hi Scott,

First of all, thank you very much for your quick answer. I hope my English is understandable because I have the language translated.
The 64 tapes is just "just for fun". With 32 bands it works for me. 64 ribbons with 2 rows each is nice too. This is supposed to be for my party room. 8 ESP32 with 127 WS2812b LEDs each are already working on the basement ceiling. That's where I wanted to use this new light game. What should I do to increase the update rate? It is worth a try. To increase the speed I switched the program to I2S. I will tell you. I had already changed the table and also recognized the problem of the same samples.
Thank you for your support.
Best wishes,
Jurgen.
